os/audio/audio.c

```diff
@@ -884,7 +885,7 @@ static void audio_callback(FAR void *handle, uint16_t reason, FAR struct ap_buff
 	break;
 
 	default: {
-		auddbg("ERROR: Unknown callback reason code %d\n", reason);
+		auddbg("%s %d\n", clog_message_str[CMN_LOG_FAILED_OP], reason);
```

Comment:
this should be invalid value on the callback not failed operation.

Reply:
I will change it.